src/Html.php

```diff
@@ -5699,8 +5699,11 @@ function(index, file) {
     public static function textarea($options = [])
     {
        //default options
+
+        $rand                   = mt_rand();
+
         $p['name']              = 'text';
-        $p['filecontainer']     = 'fileupload_info';
+        $p['filecontainer']     = 'fileupload_info' . $rand;
         $p['rand']              = mt_rand();
```

Comment:
IMHO, all options based on a random id should use the same random suffix.

```suggestion
        $p['rand']              = $options['rand'] ?? mt_rand();
        $p['filecontainer']     = 'fileupload_info' . $p['rand'];
```